scripts/src/main/resources/scripts/command/node

```diff
@@ -31,6 +31,27 @@ function doSetup() {
   fi
 }
 
+function doSetupDevon4node() {
+  if command -v devon4node &> /dev/null
+  then  
+    if [ -z "${1}" ]
+    then
+      echo "devon4node is already installed at $(command -v devon4node)"
+      exit
+    fi
+  else
+    doDevonCommand npm -q setup
+    local npm_command="${DEVON_IDE_HOME}/software/node/npm"
+    if [ ! -x "${npm_command}" ]
+    then
+      npm_command="${DEVON_IDE_HOME}/software/node/bin/npm"
+    fi
+    doDevonCommand npm -q setup
+    doDevonCommand yarn -q setup
+    doRunCommand "'${npm_command}' install -g @devon4node/cli@latest" "install devon4node"
```

Comment:
This is overcomplicated:
1. `doDevonCommand npm -q setup` is called twice
2. if you just do `doDevonCommand npm -q install -g @devon4node/cli@latest` you can replace everything else (except `doDevonCommand yarn -q setup` if you also need `yarn` as prerequisite for `devon4node`).